README.md

```diff
@@ -2,7 +2,7 @@
 ## Programmed by RokCoder
 
 Galaga is a 1981 fixed shooter arcade video game developed and published by Namco \
-This version is a tribute to the original programmed using QB64PE
+This version is a tribute to the original programmed using [QB64](https://qb64.com/)
 
```

Comment:
I'm happy for QB64 to be called out but actually wrote this using QB64PE. The following is fine -

...using [QB64PE](https://github.com/QB64-Phoenix-Edition/QB64pe) or, alternatively, [QB64](https://qb64.com/)